share/getters/shrex.go

```diff
@@ -56,23 +56,23 @@ func (m *metrics) recordNDAttempt(ctx context.Context, attemptCount int, success
 	if ctx.Err() != nil {
 		ctx = context.Background()
 	}
-	m.ndAttempts.Record(ctx, int64(attemptCount), attribute.Bool("success", success))
+	m.ndAttempts.Record(ctx, int64(attemptCount),
+		metric.WithAttributes(
+			attribute.Bool("success", success)))
 }
 
 func (sg *ShrexGetter) WithMetrics() error {
-	edsAttemptHistogram, err := meter.SyncInt64().Histogram(
+	edsAttemptHistogram, err := meter.Int64Histogram(
 		"getters_shrex_eds_attempts_per_request",
-		instrument.WithUnit(unit.Dimensionless),
```

Comment:
Removed instrument.WithUnit(unit.Dimensionless), because this is pointless to set default value and const no longer exist